jhipster/pom.xml

```diff
@@ -0,0 +1,928 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <groupId>org.springframework.boot</groupId>
+        <version>1.4.1.RELEASE</version>
+        <relativePath />
+    </parent>
+
+    <groupId>com.baeldung</groupId>
+    <artifactId>baeldung</artifactId>
```

Comment:
can we change the artifactId to something specific to project. For example, we can say **jHipster-demo**

We will also need to add the corresponding module in parent pom.xml.